Redesigned the footer and renamed the header

florisdobber opened the pull request:
Separated the footer out to its own component and redesigned it based on the footer on the NUWIT website. Added the Khoury logo in the files and renamed the NavBar to NavHeader. 

![Screenshot_2019-12-01 Code4Community](https://user-images.githubusercontent.com/44882599/69915177-1d4ae280-1444-11ea-8d47-507ed86a9d7f.png)

jackblanc commented on src/components/nav/NavFooter.vue:
```diff
@@ -0,0 +1,78 @@
+
+<script>
+export default {
+  data() {
```
We need to add a mounted() function here to load the Events, because currently that is only being done when the Events page is visited (it is being loaded from jsonplaceholder now). Look at the mounted() function in the events page for reference and add that here.

jackblanc commented on src/components/nav/NavFooter.vue:
```diff
@@ -0,0 +1,78 @@
+};
+</script>
+
+<style scoped>
```
wasn't included in the ticket, but can you add styling for mobile? For mobile, let's just have the social links show. Reference the `@media` queries in the App.vue's style tag. Play around with various viewport sizes to see how it looks (I always do this by just resizing my developer tools)

jackblanc replied:
nitpick: can we make the icons closer to the "Find us on" text?

florisdobber commented:
Moved the icons closer to the "Find us on" text, added a mounted() function to load events and optimized for mobile.
![Mobile view](https://user-images.githubusercontent.com/44882599/70330835-543a4300-180c-11ea-9af6-b676e65fd1b2.jpg)
![Desktop view](https://user-images.githubusercontent.com/44882599/70330867-6320f580-180c-11ea-9b3f-bd414aeb11dd.png)
